src/main/java/org/czekalski/petapiintegration/resourceServer/controller/AnimalController.java

```diff
@@ -0,0 +1,25 @@
+package org.czekalski.petapiintegration.resourceServer.controller;
+
+import org.czekalski.petapiintegration.resourceServer.apiResource.v1.dto.AnimalResourcesListDto;
+import org.czekalski.petapiintegration.resourceServer.service.AnimalService;
+import org.springframework.security.oauth2.client.OAuth2RestTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AnimalController {
+    private final AnimalService animalService;
+
+    public AnimalController(AnimalService animalService) {
+        this.animalService = animalService;
+    }
+
+    @GetMapping("/dogs/{cityId}/{stateId}")
```

Comment:
```suggestion
    @GetMapping("/dogs/{stateId}/{cityId}")
```
Nie pamiętam, co ostatecznie ustalaliśmy, ale tak będzie lepiej. Już mówię dlaczego. Miasto jest częścią stanu, więc naturalne jest, że jest to uszczegółowienie jakiejś lokalizacji. Możemy sobie wyobrazić, że istniałby resource `/dogs/{stateId}`, który by pokazywał psy do adopcji z danego stanu